### PR TITLE
add a data link for when a user auto signed in on fb signs out

### PR DESCRIPTION
**Pull request by NathanielBennett**
Add a data link so that we can track instances where a facebook autosignin results in an immediate signout

fb-autosign

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/identity/autosignin.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ function (
         this.welcome = function (name) {
             var msg = '<p class="site-message__message">' +
                           'Welcome ' + name + ', you’re signed into the Guardian using Facebook, or ' +
-                          '<a href="' + config.page.idUrl + '/signout"/>sign out</a>.' +
+                          '<a data-link-name="fb-auto-signout" href="' + config.page.idUrl + '/signout"/>sign out</a>.' +
```
Comment (crifmulholland) [inline static/src/javascripts/projects/common/modules/identity/autosignin.js]: Hey, I've just been told the signout events across the rest of the site say 'sign out' 

Can we please update the name to to be "fb auto | Sign out"

**Comment by NathanielBennett** [comment]
@crifmulholland: That better? I've used " : " as thats the convention used elsewhere

**Comment by phamann** [comment]
:+1:

**Comment by rich-nguyen** [comment]
:+1:

**Pull request merged**